src/Tests/Riganti.Selenium.Core.Samples.Tests/BrowsersStabilityUiTests.cs

```diff
@@ -70,15 +70,23 @@ public void SelectMethod_XPathToRoot_ExpectedException()
         }
 
         [TestMethod]
-        [ExpectedSeleniumException(typeof(InvalidSelectorException))]
         public void SelectMethod_InvalidXPathSelector_ExpectedException()
         {
-            this.RunInAllBrowsers(browser =>
+            try
             {
-                browser.NavigateToUrl("/test/ClickTest");
-                var elem = browser.Single("#span");
-                elem.Single("///***-*///@@##@šš+š++++---><<>''", By.XPath);
-            });
+                this.RunInAllBrowsers(browser =>
+                {
+                    browser.NavigateToUrl("/test/ClickTest");
+                    var elem = browser.Single("#span");
+                    elem.Single("///***-*///@@##@šš+š++++---><<>''", By.XPath);
+                });
+                throw new System.Exception("SelectMethod_InvalidXPathSelector_ExpectedException was supposed to fail!!!");
```

Comment:
This does not really make sense to me :sweat_smile: , did you intend to use Assert.Throws?

Reply:
Assert.Throw<> checks for specific exception. The mistake is incorrect place of the `throw new exc...`. I have corrected it.